### ISSUE TITLE
Broken code result when formatting GraphQL queries inside `graphql` calls

### ISSUE THREAD

**Issue by freiksenet** [prettier/prettier#2857]
**Prettier 1.7.0**
[Playground link](https://prettier.io/playground/#%7B%22content%22%3A%22graphql(schema%2C%20%60%7B%20query%20test%20%7B%20id%20%7D%7D%20%24%7Bfragment%7D%60)%22%2C%22options%22%3A%7B%22ast%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Afalse%2C%22jsxBracketSameLine%22%3Afalse%2C%22output2%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22printWidth%22%3A80%2C%22semi%22%3Atrue%2C%22singleQuote%22%3Afalse%2C%22tabWidth%22%3A2%2C%22trailingComma%22%3A%22none%22%2C%22useTabs%22%3Afalse%7D%7D)
```sh
# Options (if any):
--single-quote
```

**Input:**
```jsx
graphql(schema, `{ query test { id }} ${fragment}`)
```

**Output:**
```jsx
graphql(
  schema,
  `
    {
      query
      test {
        id
      }
    }
  ${fragment}
    {
      query
      test {
        id
      }
    }
  `
);
```

**Expected behavior:**

It's okay to ignore invalid GraphQL output or format it badly, but this actually breaks it and produces faulty code.

**Comment by freiksenet** [comment]
Related #2640

**Comment by azz** [comment]
Fixed in #2975
